tools/amd_build/build_amd.py

```diff
@@ -201,6 +201,7 @@ def remove_hcc(line: str) -> str:
                     sources.write(line)
             print(f"{hip_platform_file} updated")
 
+
```

Comment:
nit: remove?

Reply:
I need to rebase the update_ck PR once this is merged. I will make this change in that PR so we don't have to wait for CI on this one again